lib/Model/Importer/DebugInfo/PDBImporter.cpp

```diff
@@ -39,6 +39,10 @@ using namespace llvm::pdb;
 
 static Logger<> DILogger("pdb-importer");
 
+static llvm::cl::list<std::string> SearchPathPDB("search-path-pdb",
```

Comment:
Should we use the ```ImportDebugInfo``` from https://github.com/revng/revng/blob/develop/include/revng/Model/Importer/Binary/BinaryImporterOptions.h#L11?